substrate/frame/staking-async/src/tests/slashing.rs

```diff
@@ -245,12 +245,30 @@ fn subsequent_reports_in_same_span_pay_out_less() {
 				slash_fraction: Perbill::from_percent(20),
 			}],
 		);
+		assert_eq!(
+			staking_events_since_last_call(),
+			vec![Event::OffenceReported {
+				offence_era: 1,
+				validator: 11,
+				fraction: Perbill::from_percent(20)
+			}]
+		);
+
 		Session::roll_next();
 
-		// F1 * (reward_proportion * slash - 0)
-		// 50% * (10% * initial_balance * 20%)
-		let reward = (initial_balance / 5) / 20;
-		assert_eq!(reward, 10);
+		let slash = Perbill::from_percent(20) * initial_balance;
+		let reward = SlashRewardFraction::<T>::get() * slash;
+		// slash is 1000/5
+		assert_eq!(slash, 200);
+		assert_eq!(
+			staking_events_since_last_call(),
+			vec![
+				Event::SlashComputed { offence_era: 1, slash_era: 1, offender: 11, page: 0 },
+				Event::Slashed { staker: 11, amount: slash },
+			]
+		);
+		// reward is 10% of the slash
```

Comment:
Why 10% `REWARD_F1`, which was 50%, was removed; did something else replace it?

Reply:
This is explained in the PR description under the `Functional Changes` section.